swift-sdk/Internal/InternalIterableAPI.swift

```diff
@@ -72,6 +72,10 @@ final class InternalIterableAPI: NSObject, PushTrackerProtocol, AuthProvider {
         self.dependencyContainer.createAuthManager(config: self.config)
     }()
     
+    lazy var flexMessagingManager: IterableFlexMessagingManagerProtocol = {
+        self.dependencyContainer.createFlexMessageManager()
```

Comment:
I feel like we should have a consistency between "flex message" and "flex messaging". Being nit-picky.

Reply:
This one is a good catch, it's supposed to be `createFlexMessagingManager` here. I just overlooked it.